lib/src/processors/android/icons/android_generate_iclauncher_xml_processor.dart

```diff
@@ -1,17 +1,31 @@
+import 'package:flutter_flavorizr/src/parser/models/flavorizr.dart';
+import 'package:flutter_flavorizr/src/parser/models/flavors/android/adaptive_icon.dart';
 import 'package:flutter_flavorizr/src/processors/commons/string_processor.dart';
 
 class AndroidGenerateIclauncherXmlProcessor extends StringProcessor {
   AndroidGenerateIclauncherXmlProcessor({
-    required super.config,
-  });
+    required this.adaptiveIcon,
+    required Flavorizr config,
+  }) : super(config: config);
+
+  AdaptiveIcon adaptiveIcon;
```

Comment:
Can you move it above the constructor?

```suggestion
  final AdaptiveIcon adaptiveIcon;
```